diff --git a/dist/index.js b/dist/index.js
index 8b3e4da35cc1048ff486bb83740630eea6aa0f4b..aaf1814f493cd19dadad4039dd5ac74f1ac81444 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -21,7 +21,8 @@ Object.defineProperty(exports, "__esModule", { value: true });
  */
 require("@tensorflow/tfjs-core");
 // tslint:disable-next-line: no-imports-from-dist
-require("@tensorflow/tfjs-core/dist/public/chained_ops/register_all_chained_ops");
+// This creates a problem similar to https://github.com/tensorflow/tfjs/issues/4901
+//require("@tensorflow/tfjs-core/dist/public/chained_ops/register_all_chained_ops");
 const barchart_1 = require("./render/barchart");
 const confusion_matrix_1 = require("./render/confusion_matrix");
 const heatmap_1 = require("./render/heatmap");
